### PR TITLE
Some modifications in selenium machines cases

### PR DESCRIPTION
**Pull request by yunmingyang**
* Change the locator of 'Storage Pool' button in machines page
* Some enhancement

### CODE REVIEW

File: test/selenium/testlib_avocado/machineslib.py [hunk 1]
```diff
@@ -262,13 +263,19 @@ def create_vm_by_ui(self,
...
 
-        if not immediately_start:
-            self.check_box(self.wait_css('#start-vm'))
+        self.check_box(self.wait_css('#start-vm'), immediately_start)
 
         self.click(self.wait_css('#create-vm-dialog .modal-footer .btn.btn-primary', cond=clickable))
 
         self.wait_dialog_disappear()
         self.wait_css('#create-vm-dialog', cond=invisible)
+        # Record the reason of the failed
```
Comment (KKoukiou) [inline test/selenium/testlib_avocado/machineslib.py]: Useful one!

File: test/selenium/selenium-machines-storage-pool.py [hunk 2]
```diff
@@ -62,32 +63,25 @@ def testCreateDirStoragePool(self):
...
+            'tr[data-row-id="' + pool_name + '"] > td:nth-child(4)').text.split('/')
+        allocation_from_page = float(page_res[0].strip())
+        capacity_from_page = float(page_res[1].split(' ')[1])
+        # Get information from command line
+        cmd_res = self.machine.execute(
+            'sudo virsh pool-info --bytes {}'.format(name))
+        allocation_from_cmd = round(float(re.compile(r'Allocation:.*')
+                                          .search(cmd_res)
+                                          .group(0)
+                                          .split(' ')[-1]) / 1024 / 1024 / 1024,
```
Comment (KKoukiou) [inline test/selenium/selenium-machines-storage-pool.py]: You can do (1024**3) and it's more readable

File: test/selenium/selenium-machines-storage-pool.py [hunk 3]
```diff
@@ -11,8 +11,6 @@ class MachinesStoragePoolTestSuite(MachinesLib):
     :avocado: tags=machines
     """
 
-    @skipIf(os.environ.get("BROWSER") == 'edge',
```
Comment (KKoukiou) [inline test/selenium/selenium-machines-storage-pool.py]: Please add a 
Fixes #13073 line in the commit message of this commit.

File: test/selenium/selenium-machines-basic.py [hunk 4]
```diff
@@ -127,14 +127,14 @@ def testVmStatus(self):
             "fails too often, https://github.com/cockpit-project/cockpit/issues/13072")
     def testCreateVMWithISO(self):
         name = 'test_iso'
-        iso = '/home/{}.iso'.format(name + str(time.time()).split('.')[0])
```
Comment (KKoukiou) [inline test/selenium/selenium-machines-basic.py]: This is not related to the changes that are implied in the commit message title.

**Comment by martinpitt** [comment]
pep8 error:

    test/selenium/selenium-machines-basic.py:3: 'time' imported but unused

**Comment by martinpitt** [comment]
I fixed the trivial PEP-8 error and added a Closes#. Thanks @yunmingyang !
